index.html

```diff
@@ -266,11 +266,23 @@ <h3 id="types-of-ruby">Types of ruby</h3>
 
 <p>Which one to use depends on the relationship
 between the ruby and the base characters.
-Mono-ruby is used to connect ruby to a single base character,
-Jukugo-ruby is used when multiple base characters each have a corresponding ruby
+<a title="mono-ruby">Mono-ruby</a> is used to connect ruby to a single base character,
+<dfn>Jukugo-ruby</dfn> is used when multiple base characters each have a corresponding ruby
 and at the same time the whole group needs to be processed together,
-and group-ruby is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a>).
-Each is used when specified.</p>
+and <dfn>group-ruby</dfn> is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a> for examples).
```

Comment:
```suggestion
and <dfn>Group-ruby</dfn> is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a> for examples).
```

It seems that the first letters of mono/jukugo/group-ruby in this paragraph are all capitalized. If we decide this way, this one should also be capitalized.

Reply:
Thank you for catching! (I also wonder we should use capitalized for whole document...)